spec/steps/provision.fmf

```diff
@@ -97,3 +96,47 @@ example: |
             how: connect
             guest: hostname or ip address
             key: private-key-path
+
+/environment:
+    summary: Environment specification
+
+    description: |
+        As part of the ``provision`` step it is possible to
+        specify additional requirements for the testing
+        environment such as required amount of ``memory`` and
+        ``disk`` space.
+
+        FIXME: Much more details go here. For now let's start with
+        a couple of tangible examples.
```

Comment:
Example list (vagrant-libvirt), for inspiration:
https://github.com/vagrant-libvirt/vagrant-libvirt#domain-specific-options